### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by ErwanAliasr1**
Fixing my stuff. LOL.

### CODE REVIEW

File: cmd/s3_la.go [hunk 1]
```diff
@@ -30,9 +30,8 @@ func S3CmdLa(cmd *cobra.Command, args []string) {
 	output := string(execContainer(containerName, command))
 
 	if len(output) == 1 {
-		command := []string{"s3cmd", "ls"}
-		output := execContainer(containerName, command)
-		fmt.Printf("%s", output)
+		command = []string{"s3cmd", "ls"}
+		output = execContainer(containerName, command)
```
Comment (leseb) [inline cmd/s3_la.go:34]: `string(execContainer(containerName, command))`

Comment (leseb) [reply cmd/s3_la.go:34]: can't we just extract `output := string(execContainer(containerName, command))` and put it after the `if`?

File: cmd/s3_ls.go [hunk 2]
```diff
@@ -32,6 +32,5 @@ func S3CmdLs(cmd *cobra.Command, args []string) {
 	} else {
 		command = []string{"s3cmd", "ls", "s3://" + args[1]}
 	}
-	output := execContainer(containerName, command)
-	fmt.Printf("%s", output)
+	fmt.Println(execContainer(containerName, command))
```
Comment (leseb) [inline cmd/s3_ls.go:35]: don't we need to convert to a string?

**Review by leseb — CHANGES_REQUESTED** [review]

File: cmd/s3_la.go (hunk 1 above)
Comment (ErwanAliasr1) [reply cmd/s3_la.go:34]: execContainer already return a string as per func execContainer(containerName string, cmd []string) string {

File: cmd/s3_ls.go (hunk 2 above)
Comment (ErwanAliasr1) [reply cmd/s3_ls.go:35]: no ;)

**Review by leseb — APPROVED** [review]

**Pull request merged**